ansible-playbook-base/roles/common/tasks/setup_hosts_file.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I am not clear what this play does. I assume each server should have its hosts file updated with the others. I would suggest we use ansible templates and copy them over to the servers, than running a script.

Also if you can let me know what the variables are, and where they come from, we can talk about how to plumb them as well through ansible variables.